index.html

```diff
@@ -1,38 +1,196 @@
+            <li><a href="#">News</a></li>
+            <li class="summit-button">
+              <button type="button" class="summit">DDT Summit</button>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div class="header_text">
+      <h1 class="small text">"Hello Tech World"</h1>
+      <h2 class="topic" id="topic">DRONE AND DRONE TAXI 2021</h2>
+      <p class="highlight">
+        The future of passenger drones remains uncertain since this technology
+        is so new. Innovation in aerial drone technology, and in aerial
+        traffic coordination, control, and collision-avoidance could result in
+        rapid proliferation of passenger drones for civilian travel.
+      </p>
+      <p class="large">
+        <span class="venue_medium">2021.10.15(THU) ~ 16(FRI)</span>
+        <br />
+        <span class="venue">@ National Museum of Egypt Art Centre Cairo and more</span>
+      </p>
+    </div>
+  </header>
 
-<section>
+  <section class="main_program">
+    <div class="main_program_content"></div>
+    <div class="head">
+      <h2>Main Program</h2>
+      <hr class="slim" />
+    </div>
+    <div class="content-body">
+      <div class="content">
+        <span><i class="fas fa-podium-star fa-2x"></i></span>
+        <h3>Lecture</h3>
+        <p>
+          Listen to the speakers from various countries about the insight on
+          the future of drone taxi
+        </p>
+      </div>
+      <div class="content">
+        <span><i class="fas fa-drone-alt fa-2x"></i></span>
+        <h3>Drone Exhibition</h3>
+        <p>
+          Apprieciate various creation and innovative inventions, Orginised
+          from the Art Centre Cairo
+        </p>
+      </div>
+      <div class="content">
+        <span><i class="fad fa-comments-alt fa-2x"></i></span>
+        <h3>Forum</h3>
+        <p>
+          Have time to share your thoughts and opinions with experts on each
+          topic
+        </p>
+      </div>
+      <div class="content">
+        <span><i class="fas fa-chalkboard-teacher fa-2x"></i></span>
+        <h3>Workshop</h3>
+        <p>
+          Try to create your own working Drone using high-end technology
+          rather that just watching
+        </p>
+      </div>
+      <div class="content">
+        <span><i class="fad fa-project-diagram fa-2x"></i></span>
+        <h3>Collaborate</h3>
+        <p>
+          Get opportunities to network with top drone engineers from all over
+          the world, also after the summit
+        </p>
+      </div>
 
-</section>
+    </div>
+    <div class="button">
+      <button type="submit">
+        Join Drones and Drones Taxi Global Summit
+      </button>
+    </div>
+  </section>
 
+  <section class="feature_speakers flex-column-center">
+    <div class="speaker_background"></div>
+    <div class="feature-container">
+      <h2>Featured Speakers</h2>
+      <hr class="slim">
+    </div>
+    <ul class="feature-list">
```

Comment:
- [ ] Please follow the given design on this section of the page.

>A screenshot from your project.
![Screenshot2](https://user-images.githubusercontent.com/47625407/124309622-4de88600-db6b-11eb-9fd1-d634763f9722.png)


_**Please refer to the review summary and the given design link for more information.**_